src/Microsoft.Azure.WebJobs/QueueAttribute.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.WebJobs
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes")]
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{QueueName,nq}")]
-    public class QueueAttribute : Attribute
+    public class QueueAttribute : StorageAccountAttribute
```

Comment:
I think it violates #2 ... a separate Storage and StorageAccount attribute seems confusing. What's the difference between them? What's the customer value in having 2 separate attributes?